code/game/objects/structures/dropship_equipment.dm

```diff
@@ -1039,3 +1042,328 @@
 	deployed_table.layer = ABOVE_OBJ_LAYER + 0.01 //make sure its directly ABOVE the layer
 	deployed_table.loc = loc
 	icon_state = "table2-idle"
+
+//////////////// RAPPEL SYSTEM /////////////////
+
+//Rappel target selection action
+/datum/action/innate/rappel_designate
+	name = "Designate rappel point"
+	action_icon = 'icons/mecha/actions_mecha.dmi'
+	action_icon_state = "mech_zoom_on"
+	var/obj/structure/dropship_equipment/rappel_system/origin
+
+/datum/action/innate/rappel_designate/Activate()
+	var/mob/C = target
+	var/mob/camera/aiEye/remote/remote_eye = C.remote_control
+	var/turf/target_turf = get_turf(remote_eye)
+	var/area/target_area = get_area(target_turf)
+
+	if(origin.retracting) //We're already retracting a rope!
+		return
+
+	if(origin.disabled || origin.cord_damaged || target_turf.density || target_area.ceiling >= CEILING_DEEP_UNDERGROUND)
+		playsound(origin, 'sound/machines/buzz-two.ogg', 25)
+		return
+	if(origin.hooked)
+		playsound(origin, 'sound/machines/buzz-sigh.ogg', 25)
+		return
+
+	C.playsound_local(origin, 'sound/effects/binoctarget.ogg', 35)
```

Comment:
Should all these feedback sounds also have a feedback message/balloon text to actual say whats going on?

Reply:
These are for the person in the camera actually designating the spot, who can't see balloon alerts on the system & won't be able to reliably see balloon alerts on NVG. I get your point, though - would making the feedback sounds local to the pilot be enough?